packages/react/src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -247,6 +249,49 @@ const BarChartCard = ({
     [categoryDataSourceId, chartData, timeDataSourceId, type, values]
   );
 
+  const previousTick = useRef();
+  dayjs.locale(locale);
+
+  const maxTicksPerSize = useMemo(() => {
```

Comment:
nit: perhaps this could be a shared custom hook if it is used in multiple places. Not sure it is worth the trouble..

Reply:
I thought about it, but not sure it's worth the time. I think these two places are the only place it's used, and I think, IIRC, the BarChartCard is supposed to replace the TimeSeriesCard at some point, but I could be making that up...

Reply:
On the quarter interval. I'm honestly not sure. It matches the single date that TimeSeriesCard has, but it could be a bug there, too. @scottdickerson what's the expectation for quarterly intervals?

Reply:
@scottdickerson any input here?

Reply:
@kevinsperrine let's fix the conflicts and we will merge this.